tools/explorer/pkg/stellar/stellar.go

```diff
@@ -32,10 +32,11 @@ type (
 	// Wallet is the foundation wallet
 	// Payments will be funded and fees will be taken with this wallet
 	Wallet struct {
-		keypair *keypair.Full
-		network string
-		assets  map[Asset]struct{}
-		signers Signers
+		keypair        *keypair.Full
+		network        string
+		assets         map[Asset]struct{}
+		signers        Signers
+		minimumBalance float64
```

Comment:
calculating the minimum balance is very cheap computationally, so lets just store it in a function on the wallet. This also means that we don't have to change anything in the future if we would decide to load more signers // assets at runtime

Reply:
Actually, a rounding error cannot occur here. Since signers is always an even number?